common/src/main/java/io/druid/math/expr/Parser.java

```diff
@@ -148,30 +148,14 @@ public void visit(Expr expr)
 
   public static Expr.ObjectBinding withMap(final Map<String, ?> bindings)
   {
-    return new Expr.ObjectBinding()
-    {
-      @Override
-      public Number get(String name)
-      {
-        Number number = (Number)bindings.get(name);
-        if (number == null && !bindings.containsKey(name)) {
-          throw new RuntimeException("No binding found for " + name);
-        }
-        return number;
-      }
-    };
+    return bindings::get;
```

Comment:
Because rather than treating expressions referencing missing columns as errors, I'd like to treat them as if they were reading nulls. Just doing `bindings::get` achieves that since `bindings.get(nonexistentColumnName) == null`